### PR TITLE
Reboot

### PR DESCRIPTION
**Pull request by hrag**
Initial first (gigantic) React rendition of StyleDot

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
 language: node_js
+
 node_js:
-  - "10"
```
Comment (blopker) [inline .travis.yml]: You should probably use 10 here.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,82 @@
-# Style
-[![Build Status](https://travis-ci.org/23andMe/style.svg?branch=master)](https://travis-ci.org/23andMe/style)
+# buble-react-rollup-starter [![Build Status](https://travis-ci.org/yamafaktory/buble-react-rollup-starter.svg?branch=master)](https://travis-ci.org/yamafaktory/buble-react-rollup-starter) [![npm version](https://img.shields.io/npm/v/buble-react-rollup-starter.svg?style=flat)](https://www.npmjs.com/package/buble-react-rollup-starter) [![Standard - JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
```
Comment (blopker) [inline README.md]: You removed our beautiful readme!

File: src/guide/scss/_normalize.scss [hunk 3]
```diff
@@ -0,0 +1,447 @@
+/*! normalize.css v7.0.0 | MIT License | github.com/necolas/normalize.css */
```
Comment (blopker) [inline src/guide/scss/_normalize.scss]: There's a new version of this.

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,3309 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (blopker) [inline yarn.lock]: Why is there a yarn file and a package-lock?

File: .travis.yml (hunk 1 above)
Comment (hrag) [reply .travis.yml]: Updated in my current working branch feature/STYLE-2.

**Pull request merged**